API/hermes/CMakeLists.txt

```diff
@@ -93,8 +93,8 @@ elseif("${CMAKE_CXX_COMPILER_ID}" MATCHES "MSVC")
 endif()
 
 set_target_properties(libhermes PROPERTIES
-  COMPILE_FLAGS "${compile_flags}"
-  LINK_FLAGS "${link_flags}"
+  COMPILE_FLAGS "${compile_flags} -Wl,--hash-style=both"
```

Comment:
This was needed to get the libs loaded on older platforms .. but it's not specific to intl code.. Hence, there is something i'm missing and it is probably not the best ways to do it ..